lib/time.ts

```diff
@@ -28,9 +28,12 @@ export const useSiteTime = (
     latitude && longitude ? ['/timezone', longitude, latitude] : null,
     () => fetchTimeZone(latitude, longitude)
   );
+
   const [currentTime, setCurrentTime] = useState(dayjs().tz(timezone));
+  const [tomorrow, setTomorrow] = useState(currentTime.add(1, 'day'));
 
   useEffect(() => setCurrentTime(dayjs().tz(timezone)), [timezone]);
+  useEffect(() => setTomorrow(currentTime.add(1, 'day')), [currentTime]);
```

Comment:
Since `tomorrow` is always just a value computed from `currentTime`, it doesn't have to be a state variable.

You _could_ `useMemo` this, but maybe not necessary.
```suggestion
  const tomorrow = currentTime.add(1, 'day');

  useEffect(() => setCurrentTime(dayjs().tz(timezone)), [timezone]);
```